app/components/chart/footer.js

```diff
@@ -74,28 +77,41 @@ var TidelineFooter = React.createClass({
             onChange={this.props.onClickLines} /> Lines
         </label>
       </div>
-      );
-    
+    );
 
-    
-    var rightSide = this.props.chartType === 'weekly' ? showValues :
-      this.props.chartType === 'modal' ? modalOpts : null;
-    
+    var rightSide = null;
+    var bgDataToggle = null;
+
+    if (this.props.chartType === 'weekly') {
+      rightSide = showValues;
+    } else if (this.props.chartType === 'modal') {
+      if (this.props.showingSmbg) {
+        rightSide = modalOpts;
+      }
+      bgDataToggle = (
+        <TwoOptionToggle
+          left={{ label: 'BGM', state: this.props.showingSmbg }}
+          right={{ label: 'CGM', state: this.props.showingCbg }}
```

Comment:
Is there a reason that we're storing this state in two props as opposed to the second just being `!this.props.showingSmbg` or something similar? It just feels strange to have essentially one piece of information stored as two things that will always be the inverse of one another.